internal/config/instance.go

```diff
@@ -231,6 +231,10 @@ func (i *Instance) ReadInConfig() error {
 
 	err = pl.WaitForLock(5 * time.Second)
 	if err != nil {
+		lockedErr := &lockfile.AlreadyLockedError{}
+		if errs.Matches(err, &lockedErr) {
```

Comment:
```suggestion
		if errs.Matches(err, &lockfile.AlreadyLockedError{}) {
```

The whole point of `errs.Matches` is that you don't need to do this silly trickery. If the above doesn't work you're probably not constructing the error properly.